### PR TITLE
No Password Policy

### PR DESCRIPTION
**Pull request by RRudder**

### CODE REVIEW

File: submissions/description/insufficient_security_configurability/no_password_policy/template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+## Business Impact
+
+Having no password policy can result in reputational damage for the business through the impact to customers’ trust as they could believe that the business doesn’t take their account security seriously or trust that their data within will remain secure.
+
+## Steps to Reproduce
+
+1. Use a browser to navigate to: {{URL}}
+1. Attempt to login
+1. Observe that the application allows the use of weak passwords, such as `a`, `password`, password123`
```
Comment (evildaemond) [inline submissions/description/insufficient_security_configurability/no_password_policy/template.md:15]: This should be updated as no password policy means quite literally 1 char, so I've updated this to reflect that

 1. Observe that the application allows the use of weak passwords, such as `a`

**Review by evildaemond — CHANGES_REQUESTED** [review]
1 change then LGTM

File: submissions/description/insufficient_security_configurability/no_password_policy/template.md (hunk 1 above)
Comment (RRudder) [reply submissions/description/insufficient_security_configurability/no_password_policy/template.md]: Thanks for the pick up

**Review by evildaemond — APPROVED** [review]
LGTM

**Pull request merged**